### PR TITLE
Adopt Docker guidelines for documentation and grammar conventions #40

### LINKED ISSUE #40: Adopt Docker guidelines for documentation and grammar conventions
Perhaps we can refer to this instead of defining our own? 
https://docs.docker.com/opensource/doc-style/

### PR DESCRIPTION
**Pull request by damithc**
Fixes #40 

```
Instead of writing our own documentation style and grammar
guidelines, refer to those used by Docker.
```

### CODE REVIEW

**Comment by CanIHasReview** [comment]
# v1

@damithc submitted v1 for review.

* [v1 1/1] [Adopt Docker documentation and grammar guidelines](https://github.com/oss-generic/process/pull/65/commits/66943ba5f154dd2689367c1d762a012c2137f955)

([:books: Archive](https://github.com/oss-generic/process/compare/7d75bf6a6ab1ed5559d28c67a61c04e43ad910e9...66943ba5f154dd2689367c1d762a012c2137f955))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/oss-generic/process.git refs/pr/65/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by CanIHasReview** [comment]
# v2

@damithc submitted v2 for review.

* [v2 1/1] [Adopt Docker documentation and grammar guidelines](https://github.com/oss-generic/process/pull/65/commits/3fbc5299718473b6bacc51f6becceb176a25a89f)

([:books: Archive](https://github.com/oss-generic/process/compare/7d75bf6a6ab1ed5559d28c67a61c04e43ad910e9...3fbc5299718473b6bacc51f6becceb176a25a89f)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/oss-generic/process/compare/36eb7164141598a56bb50236dff1d377381b9f9c~1...36eb7164141598a56bb50236dff1d377381b9f9c))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/oss-generic/process.git refs/pr/65/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: docs/FormatsAndConventions.md [hunk 1]
```diff
@@ -99,6 +99,16 @@ Commits of a PR should be organized to match the following requirements:
 * Use lowerCamelCase (similar to java methods) whenever possible. e.g. `testData`
 * Prefer plurals if the folder contains multiple items of same type e.g. `docs` instead of `doc`
 
+## English
+
+Follow [Docker's documentation style and grammar conventions](https://docs.docker.com/opensource/doc-style/)
+if the same is not covered by our own conventions (for example, we have our own PR title convention
+that should take precedence over that of Docker's).
+
+Use American English spelling.
```
Comment (acjh) [inline docs/FormatsAndConventions.md]: Should these 2 points be list items like the other sections?

**Comment by CanIHasReview** [comment]
# v3

@damithc submitted v3 for review.

* [v3 1/1] [Adopt Docker documentation and grammar guidelines](https://github.com/oss-generic/process/pull/65/commits/01369220f94864217d2bcd5cd63a451a486454ed)

([:books: Archive](https://github.com/oss-generic/process/compare/7d75bf6a6ab1ed5559d28c67a61c04e43ad910e9...01369220f94864217d2bcd5cd63a451a486454ed)) ([:chart_with_upwards_trend: Interdiff between v2 and v3](https://github.com/oss-generic/process/compare/238445b70832c1f73ad86e40c00390ee1fe33b0a~1...238445b70832c1f73ad86e40c00390ee1fe33b0a))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/oss-generic/process.git refs/pr/65/3/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by damithc** [comment]
Going to merge soon.

**Pull request merged**